lib/nerves_bootstrap.ex

```diff
@@ -44,8 +44,7 @@ defmodule Nerves.Bootstrap do
   def add_aliases(aliases) do
     aliases
     |> append("deps.get", "nerves.deps.get")
-    |> prepend("deps.precompile", "nerves.precompile")
-    |> append("deps.loadpaths", "nerves.loadpaths")
+    |> prepend("deps.loadpaths", "nerves.loadpaths")
```

Comment:
How critical is this append/prepend swap for `deps.loadpaths`? For example, if someone has a project that isn't using `Nerves.Bootstrap.add_aliases/1`, how sad are they?

Reply:
Its not critical to the point that it will not break existing projects. It is only meaningful when paired with the logic that clears the dependency cache to fix issues with `System.get_env` in dependency `mix.exs` calls and any `rebar.config.scripts` that call `os:getenv/1` to ensure they are reevaluated after `Nerves.Env.bootstrap/0` but before any of them are compiled.